### DISCUSSION TITLE
Power Clamp / PC321 / detected as ir_remote_sensors

### DISCUSSION THREAD

**Discussion by maressl** [make-all/tuya-local#1302] [Q&A]
I have problem that my power clamp PC321 are detected as ir_remote_sensors ?!?

    "model": "PC321",
    "product_id": "r64livttuufctble",
    "product_name": "PC321-TY-WB-IPEX"

If i delete file ir_remote_sensors.yaml then device is unknown.
In file pc321ty_energy_meter.yaml its on top of the file defined:

name: Power clamp meter
products:
  - id: r64livttuufctble --> its same as product Id !?!?
    name: PC321-TY

Why is power clamp not detected? What must i do?

**Comment by make-all** [comment]
Look at the logs.
